controllers/watcherapi_controller.go

```diff
@@ -274,28 +284,74 @@ func (r *WatcherAPIReconciler) generateServiceConfigs(
 	if err != nil {
 		return err
 	}
+
+	databaseAccount := string(secret.Data[DatabaseAccount])
+	db, err := mariadbv1.GetDatabaseByNameAndAccount(ctx, helper, watcher.DatabaseCRName, databaseAccount, instance.Namespace)
+	if err != nil {
+		return err
+	}
 	// customData hold any customization for the service.
 	// NOTE jgilaber making an empty map for now, we'll probably want to
```

Comment:
this comment can be removed now that we are actually implementing custom config, so it's up to date